nav2_bringup/params/nav2_params.yaml

```diff
@@ -135,46 +135,45 @@ controller_server:
       yaw_goal_tolerance: 0.25
     # DWB parameters
     FollowPath:
-      plugin: "dwb_core::DWBLocalPlanner"
```

Comment:
@SteveMacenski  I think I noticed some unrelated code got added in inadvertently :sweat_smile: